index.html

```diff
@@ -206,5 +206,83 @@ <h4>Skills</h4>
         
     </section>
 </section>
+
+<footer>
```

Comment:
- [ ] Let's make sure our background covers the entire view width, you could see it doesn't cover width totally as per design, could you fix that, please? 

Your Implementation | Figma Design
:----:|:----: 
![image](https://user-images.githubusercontent.com/87689547/149036326-b9c030b5-d243-4405-89fd-cd82bcc4895d.png) | ![image](https://user-images.githubusercontent.com/87689547/149036351-a41a3a62-ece5-4d80-88da-3333036b367b.png)

Reply:
That's the default view from a browser. Nothing can be done about it